Makefile

```diff
@@ -34,19 +36,16 @@ verify-codegen:
 
 .PHONY: image
 image:
-	docker build -t $(IMAGE):latest .
+	docker build -t $(IMAGE):$(DEV_VERSION) .
 
 .PHONY: image-release
 image-release:
 	docker build -t $(IMAGE):$(VERSION) .
 
 .PHONY: push
 push:
-ifeq ($(AWS_ACCOUNT),)
-	$(error AWS_ACCOUNT is not set)
-endif
-	docker tag $(IMAGE):latest $(REPO):latest
-	docker push $(REPO):latest
+	docker tag $(IMAGE):$(DEV_VERSION) $(REPO):$(DEV_VERSION)
```

Comment:
Why remove the AWS_ACCOUNT check?  Its required in by REPO.

Reply:
added it back. should we add same check to push-release as it also depends on REPO?